CHANGES.md

```diff
@@ -42,6 +52,13 @@ To be released.
  -  Added IPv6 support to `Libplanet.Stun.StunAddress`. [[#267], [#271]]
  -  `IStore.GetBlockStates()` became able to return `null` to represent an absence
     of states (i.e., incomplete states).  [[#272], [#285]]
+ -  `Swarm` became to broadcast staged `Transaction`s periodically
+     so that game apps no more need to maintain their own thread to
+     broadcast staged transactions.  [[#274], [#297]]
+ -  Currently, Swarm sent `GetTxs` with an empty `txIds` when it receives a
+    `TxIds` message and if it does not have unknown `TxId`s.  Changed so that
+    the `GetTxs` message is not sent when there are no unknown `TxId`s.
+    [[#297]]
```

Comment:
Thanks for elaboration.  In changelog, don't a present tense or words like “current” denote the version after a change made, because people read this after we release a new version?  To me it'd better be:

```markdown
 -  Previously, `Swarm` had sent an empty `GetTxs` message when it receives
    an empty `TxIds` from peers, and it had made the network waste bandwidth for
    unnecessary messages.  `Swam` became to no more send such empty `GetTxs`.
    [[#297]]
```

Reply:
I updated the changelog.